onnxruntime/core/common/profiler.h

```diff
@@ -78,6 +78,17 @@ class Profiler {
                              const std::initializer_list<std::pair<std::string, std::string>>& event_args = {},
                              bool sync_gpu = false);
 
+  void EndTimeAndRecordEvent(EventCategory category,
+                             const std::string& event_name,
+                             const TimePoint& start_time, const TimePoint& end_time,
+                             const std::initializer_list<std::pair<std::string, std::string>>& event_args = {},
+                             bool sync_gpu = false);
+
+  void EndTimeAndRecordEvent(EventCategory category,
+                             const std::string& event_name,
+                             long long dur, long long ts,
```

Comment:
What do 'dur' and 'ts' represent here? Both are of type 'long long' and it's easy to get confused and mess up things. Needs clear documentation.

Reply:
'dur' is the duration of the running time of an op, while 'ts' is the time difference between profiler start time and op start time.

Reply:
Can you document and rename these parameters appropriately that reflects the meaning esp. now that these methods are exposed as public methods?